docs/how-to/DataScienceScripts.md

```diff
@@ -0,0 +1,154 @@
+# Data Science Scripts - Foundational Structure: How to start drafting the DS scripts with respect to Model Factory Architecture
+
+## Context and Scope
+
+Data scientists perform exploratory data analysis (EDA) to understand the training data, and then perform experiments to create machine learning code that tunes a base model for the hypothesis based on the analysis of the training data. Data scientists can perform this work on their own workstation, on a Data Science Virtual Machine (DSVM), or on an Azure Machine Learning Compute instance.
+
+Data Scientists need a flexible development environment with some engineering fundamentals such as main branch protection for PR review as well as a simple CI pipeline with python linting, secrets checking, markdown linting, etc. This environment let's the data science experiment with models running on Azure Machine Learning compute without impacting production code.
```

Comment:
"This environment let's the data science experiment"

should be

"This environment lets the data scientist experiment"

Reply:
Thanks @marshallbentley fixed it